packages/prisma/migrations/20220525182228_cal_video_preinstalled/migration.sql

```diff
@@ -0,0 +1,3 @@
+insert into "App" ("slug", "dirName", "updatedAt", "categories") 
+values ('daily-video', 'dailyvideo', CURRENT_TIMESTAMP, '{video}') 
+on conflict do nothing
```

Comment:
This covers having Cal Video installed by default. Follow-up changes will tackle removing globally installed apps to also be able to uninstall them, no special treatment.